src/constants.js

```diff
@@ -1,10 +1,10 @@
 export const airnoteProductUID = "product:org.airnote.solar.v1";
+export const radnoteProductUID = "product:org.airnote.solar.rad.v1";
```

Comment:
While you’re in here you might want to capitalize all these for consistency. Not sure why these are randomly camel cased.